bucket/gosec.json

```
@@ -0,0 +1,24 @@
+{
+    "version": "2.13.1",
+    "description": "Inspects source code for security problems by scanning the Go AST.",
+    "homepage": "https://github.com/securego/gosec",
+    "license": "Apache2.0",

```

```suggestion
    "license": "Apache-2.0",
```

License should be a valid SPDX identifier: https://spdx.org/licenses/